extension/src/popup/components/hardwareConnect/LedgerSign/index.tsx

```diff
@@ -134,12 +145,15 @@ export const LedgerSign = () => {
           </div>
         </div>
         <div className="LedgerSign__bottom">
-          <LedgerErrorBlock error={connectError} />
+          {isDetectBtnDirty && <LedgerErrorBlock error={connectError} />}
```

Comment:
hide the connect error before user clicks "detect"

https://stellarorg.atlassian.net/browse/WAL-388